packages/commerce-sdk-react/src/auth/index.test.ts

```diff
@@ -39,11 +39,6 @@ jest.mock('../utils', () => ({
     onClient: () => true
 }))
 
-test('injectAccessToken', () => {
```

Comment:
If we remove the test, Do we also want to remove the `injectAccessToken` logic?

Reply:
Ok, I see it was removed here: https://github.com/SalesforceCommerceCloud/pwa-kit/pull/964/files#diff-4f3bb7407698d0a81a28ff00462f1589482fa8e984b6b19622286c5f6db36bb1L345